### DISCUSSION TITLE
GitHub Docs theme

### DISCUSSION THREAD

**Discussion by mhowes-veracode** [community/community#20688] [Discussions] [answered]
What theme is used on the GitHub Docs site: https://docs.github.com/en

Is this a Jekyll theme and is it publicly available for others to use?

**Comment by MrSarno** [comment]
I don't believe the theme used on GitHub Docs has been made available anywhere.

GitHub Docs appears to be created using ReactJS & NextJS. There's a few giveaways, such as finding "data-reactroot" in div tags.
 
 
Hope this helps! :)

**Comment by The-Real-Thisas** [comment]
Hi @mhowes-veracode 👋

Unfortunately as mentioned by @MrSarno github docs is not using the default github pages themes, instead it is a custom react site developed using their internal design system [primer](https://primer.style/).

However if you are really set on recreating the style of the docs website you can do so by using [primer](https://primer.style/) to create a custom theme for one of the many documentation generators such as [docusaurus](https://docusaurus.io/). 

Link to Primer: https://primer.style/
Link to Docusaurus: https://docusaurus.io/

Hope you find what your looking for,
Thisas

**Comment by mhowes-veracode** [comment, marked as answer]
Thank you @MrSarno and @The-Real-Thisas !
